e2e-tests/playwright.config.ts

```diff
@@ -62,6 +62,25 @@ export default defineConfig({
         '**/playwright/e2e/verify-tls-config-with-external-postgres-db.spec.ts',
       ],
     },
+    {
+      name: 'showcase-aks',
```

Comment:
any specific idea to have new project? I guess we want to run all tests on AKS.

Reply:
I've got a two reasons for that:
- we can start running tests on AKS before we do the manual setup that is needed for running all the tests ([RHIDP-2128](https://issues.redhat.com/browse/RHIDP-2128) demands installation and Azure integration tests only)
- I believe that some tests might break only on AKS and it is good to have an option to disable them only for AKS

Reply:
Agree, lets keep this way then.